trestle/core/commands/version.py

```diff
@@ -30,7 +33,44 @@ class VersionCmd(CommandBase):
 
     name = 'version'
 
+    def _init_arguments(self) -> None:
+        self.add_argument('-n', '--name', help='Name of the OSCAL object (folder name)', default=None, type=str)
+        self.add_argument(
+            '-t',
+            '--type',
+            help='Name of the OSCAL type (catalog, profile, component-definition, system-security-plan, etc)',
+            default=None,
+            type=str
+        )
```

Comment:
I would say "Name of the OSCAL model" (and not mention folder name).  'Object' sounds lower level than model - and this only deals with what we refer to as OSCAL models.  None of the CLI commands mentions folder name, so this would stand out in that regard.  Models have a type and they are stored in trestle somehow by name - so just refer to the model name and type.

And on line 41 I would just say "Type of the model being queried: (catalog ...)"  This emphasizes the -t as type, and I don't think we need to say "name of the type".

Reply:
done